source/agora/node/RunNode.d

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+
+    Boots up a node that listen for network requests and blockchain data
+
+    This is called from the main or CLI.
+    The initialization process of the node is then completed.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.node.RunNode;
```

Comment:
Is there a reason it's in own module, as opposed to e.g. `Node` ?

Reply:
To resolve @Geod24's request to reuse `runNode()`, the main.d of agora has been modified.
Wasn't this what you wanted?

Reply:
How about this? `agora.utils.RunNode`

Reply:
Why can't it just be moved to the existing `agora.node.Node` ?

Reply:
Yes, that sounds good, then I'll add it at the end.